src/olympia/scanners/admin.py

```diff
@@ -323,3 +323,23 @@ class ScannerRuleAdmin(admin.ModelAdmin):
 
     list_display = ('name', 'scanner', 'action', 'is_active')
     list_filter = ('scanner', 'action', 'is_active')
+    fields = (
+        'created',
+        'modified',
+        'name',
+        'scanner',
+        'action',
+        'definition',
+        'matched_results_link',
+        'is_active',
+    )
+    readonly_fields = ('created', 'modified', 'matched_results_link',)
+
+    def matched_results_link(self, obj):
```

Comment:
Didn't think it was worth a code comment, but FYI I didn't use `zadmin.admin.related_content_link` because I wanted more flexibility around the url and possibly the text/queryset later as well.